glean-core/ios/Glean/Config/Configuration.swift

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/// The Configuration struct describes how to configure Glean
+///
+/// - parameters:
```

Comment:
Let's go the markdown route and also use the same phrasing as for the Rust code (unless there's guidelines for docs we should adhere to?)

```suggestion
/// # Arguments
```

Reply:
I think the "Quick Help" looks for the `- parameters:` specifically

Reply:
Well, then we gotta keep it :)